Source/WebCore/css/typedom/transform/CSSMatrixComponent.h

```diff
@@ -40,7 +40,7 @@ template<typename> class ExceptionOr;
 class CSSMatrixComponent : public CSSTransformComponent {
     WTF_MAKE_ISO_ALLOCATED(CSSMatrixComponent);
 public:
-    static Ref<CSSTransformComponent> create(Ref<DOMMatrixReadOnly>&&, std::optional<CSSMatrixComponentOptions>&&);
+    static Ref<CSSTransformComponent> create(Ref<DOMMatrixReadOnly>&&, CSSMatrixComponentOptions&&);
```

Comment:
How about we use ` = { }` for the last parameter and stop passing `{ }` explicitly at the call sites?